### DISCUSSION TITLE
Return Payment

### DISCUSSION THREAD

**Discussion by awais773** [community/community#69494] [Copilot Conversations]
### Select Topic Area

General

### Body

I subscribe to the Copilot for free but I do not use and I am not interested in Copilot,
Copilot auto deducts the payment from my account so please return the amount to my same account please accept my request,
Thanks

**Comment by Akash1134** [comment]
Hi @awais773 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
